mainwindow.cpp

```diff
@@ -638,6 +663,14 @@ QStringList MainWindow::saveAttackParameters()
     if (m_ui->checkBox_EnvironmentVar->isChecked()) {
         m_settings.setValue("environmentVariables", m_ui->lineEdit_EnvironmentVar->text());
     }
+    // Save unselected rows
+    QList<QVariant> unselectedRows;
```

Comment:
We can't directly do QList<int> = m_settings.value("unselectedRows").toList(); but I'll come up with a workaround.
error conversion from 'QList<QVariant>' to non-scalar type 'QList<int>' requested
     QList<int> unselectedRows = m_settings.value("unselectedRows").toList();
                                                                           ^